snapcraft/cli/_errors.py

```diff
@@ -43,7 +42,6 @@
 # - annotate the part and lifecycle step in the message
 # - add link to privacy policy
 _MSG_TRACEBACK_PRINT = "Sorry, an error occurred in Snapcraft:"
```

Comment:
it would be great to see a follow-up PR that concats the error message and this on the same line